posthog/templates/email/first_ingestion_reminder.html

```diff
@@ -19,13 +19,12 @@ <h3>Welcome, {{ first_name }}!</h3>
             <a
                 href="https://posthog.com/docs/integrate/third-party/segment?{{ utm_tags }}"
                 ><strong>third-party platform</strong></a
-            >. It’s easy. Our docs will show you how!
+            >. It’s easy. <a href="https://posthog.com/docs">Our docs</a> will show you how!
         </li>
         <li class="mb">
             <strong>If you're <i>not</i> an engineer:</strong>
 
-            <a href="{% absolute_uri '/organization/settings' %}?{{ utm_tags }}" target="_blank"><strong>Invite a colleague with code access</strong></a> so
-            they can set it up for you. PostHog works best with friends anyway.
+            <a href="{% absolute_uri '/organization/settings' %}?{{ utm_tags }}" target="_blank"><strong>Invite a colleague with code access</strong></a> to set it up for you. PostHog works best with friends anyway!
```

Comment:
TBH I had to give it a hard think to figure out that "code access" means "able to update your product's code" and not "has some sort of secret code" – can we reword this for clarity?

Reply:
```suggestion
            <a href="{% absolute_uri '/organization/settings' %}?{{ utm_tags }}" target="_blank"><strong>Invite a colleague who can make changes to your product</strong></a> to set it up for you. PostHog works best with friends anyway!
```